pizzapi/utils.py

```diff
@@ -1,11 +1,13 @@
 import requests
 import xmltodict
-
+from time import sleep
 
 # TODO: Find out why this occasionally hangs
 def request_json(url, **kwargs):
     r = requests.get(url.format(**kwargs))
     r.raise_for_status()
+    #Testing a sleeper to see if this function is being to aggressive
+    sleep(2)
```

Comment:
I'm pretty sure dominos' rate limiting is tied to the hour, so sleeping for two seconds shouldn't have any effect.

Reply:
yes i noticed this as well.  i mean to remove that.  it seems like there was a limit of 5 or so requests every 10 minutes when i was testing.  maybe more maybe less
i'll get this updated as well.